Sources/Purchasing/Configuration.swift

```diff
@@ -174,6 +177,20 @@ import Foundation
             return self
         }
 
+        /// Set `showStoreMessagesAutomatically`. Enabled by default.
+        /// If enabled, if the user has billing issues, has yet to accept a price increase consent or
+        /// there are other messages from StoreKit, they will be displayed automatically when the app is initialized.
+        ///
+        /// If you want to disable this behavior so that you can customize when these messages are shown, make sure
+        /// you configure the SDK as early as possible in the app's lifetime, otherwise messages will be displayed
```

Comment:
Can you explain this part? I don't think that's important, because the SDK won't display them until it's initialized?

Reply:
By default, if you don't do anything, iOS will show the messages. iOS stops showing these messages if you start listening for messages, but if you don't do it early enough in the app's lifecycle, it's possible iOS may have already shown some messages. This is why they recommend setting up the listener as early as possible: `If you want to defer or suppress App Store messages, set up the message listener in your app when your app launches`.

https://developer.apple.com/documentation/storekit/message